gammapy/maps/wcs.py

```diff
@@ -541,11 +541,13 @@ def pix_to_idx(self, pix, clip=False):
                 else:
                     np.clip(idxs[i], 0, self.axes[i - 2].nbin - 1, out=idxs[i])
             else:
+                # TODO: indices out of range are set to -1 here. Maybe the behaviour
```

Comment:
This sounds like an important question that we should decide / address soon.
Remove this TODO and open a GH issue to discuss it tomorrow int he call?

(ideally with a minimal example to show what this is about).

Reply:
Done, see #1440.